melissa/actions/define_subject.py

```diff
@@ -4,13 +4,21 @@
 # Melissa
 from melissa.tts import tts
 
-WORDS = {'define_subject': {'groups': ['define']}
+WORDS = {'define_subject': {'groups': [['define'],['what', 'is']]}
          }
 
 
 def define_subject(speech_text):
     words_of_message = speech_text.split()
-    words_of_message.remove('define')
+    try:
```

Comment:
>so should it be defined as 3 seperate functions that remove words and pass to main wiki lookup

yes.

> can keywords be set to only respond when at beginning of sentance

while it is possible, IMO maybe not. the first time the function is created, it is under assumption that the keyword will be removed and the words before the keyword will not affect the request to wiki lookup. 

until user not file bug request about this, just use simple assumption like before. 

or maybe @tanay1337 have other opinion?